LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 Netherlands eScience Center
+   Copyright [yyy] [name of copyright owner]
```

Comment:
This is the license for the template, why replace this with a placeholder?

Reply:
Because this is the appendix to the license that tells other people how to apply it to their own work. They don't do that by claiming ownership of the eScience Center.

The way it works is that the license itself is completely generic. You use it by putting a license grant statement somewhere, like in the README, that says that you own the software and that it can be used under the terms of the Apache license. The license itself is a separate file (LICENSE in this case, COPYING often for the GPL), which contains the generic license text, and also an instruction on how to apply it to your own work in case you like what you see.

So, in the README.rst for the template, there should be a note saying that it's copyright NLeSC and licensed under the Apache license, and LICENSE is then the generic Apache License text.

The template generates it in the same way with this PR, at least for the Apache License and the GPL. The BSD, MIT and ISC licenses are very short, and contain references to the copyright owner (because they were never really designed to be reused, so you have to copy-paste and modify), so they're usually included in a README, and we also put them into LICENSE so that GitHub will pick them up.

Reply:
I didn't know that. Thanks for explaining.